tests/publiccloud/check_registercloudguest.pm

```diff
@@ -148,6 +148,52 @@ sub check_instance_unregistered {
     }
 }
 
+sub new_registration {
+    record_info('Starting registration...');
+    $instance->ssh_script_retry(cmd => "sudo ${path}/registercloudguest $regcode_param", timeout => 300, retry => 3, delay => 120);
```

Comment:
**Attention**: `$path` is only defined local in `run` only, here not available.
Same, in other similar use here below.

Reply:
As an alternative to pass this parameter, move it in module-header, as common to the module.

Reply:
As an alternative to pass this parameter, move it up in module-header, as common to the module. @asmorodskyi wdyt ?

Reply:
Indeed it is. I initially passed it to each function but that quickly became boilerplate-y. I figured I'd discuss whether it made sense to make it module-wide variable or to pass it to each routine.

Reply:
agree with @m-dati  just made it global

Reply:
Global it is.